pram

```diff
@@ -218,6 +224,19 @@ main() {
 			continue
 		fi
 
+		if ${interactive} ; then
+			local answer
+			read -p "Do you want to merge this? (y/n) " answer
+			case ${answer} in
+				y)
```

Comment:
How about allowing empty string too? I suppose we should default to yes. Also make it `(Y/n)` in the prompt then.